packages/extension-ui/src/components/Address.tsx

```diff
@@ -104,10 +108,15 @@ function Address ({ address, className, children, genesisHash, name, actions }:
           />
           <Info>
             <Name>{name || (account && account.name) || '<unknown>'}</Name>
-            <FullAddress>{formatted || '<unknown>'}</FullAddress>
-            {chain?.genesisHash && (
-              <Banner>{chain.name}</Banner>
-            )}
+            <CopyAddress>
+              <FullAddress>{formatted || '<unknown>'}</FullAddress>
+              {chain?.genesisHash && (
+                <Banner>{chain.name}</Banner>
+              )}
+              <CopyToClipboard text={(formatted && formatted) || ''} >
+                <Svg src={CopyImg} onClick={(): void => show('Copied')}/>
```

Comment:
Do this via a `useCallback` (and since touching, we may want to turn the other functions into them as well, there seems to be only 1)